### PR TITLE
fix: Configure kokoro jobs for functions-framework-java release 

### PR DESCRIPTION
**Pull request by kappratiksha**
https://github.com/GoogleCloudPlatform/functions-framework-java has three different packages which use different kokoro jobs. Adding the configuration for the same

### CODE REVIEW

**Review by chingor13 — CHANGES_REQUESTED** [review]

File: releasetool/commands/tag/java.py [hunk 1]
```diff
@@ -55,6 +55,9 @@ def kokoro_job_name(upstream_repo: str, package_name: str) -> Union[str, None]:
     if repo_short_name in java_framework_release_pool_repos:
         return f"cloud-java-frameworks/{repo_short_name}/stage"
 
+    if repo_short_name == "functions-framework-java" and package_name in functions_framework_java_packages:
+        return f"functions-framework/java/{package_name}/release"
```
[the remark this answers is not in the document]
Comment (chingor13) [reply releasetool/commands/tag/java.py:59]: Also, note that we do have a tiny test suite for the autorelease trigger behavior: https://github.com/googleapis/releasetool/blob/master/tests/test_autorelease_trigger.py

[the remark this answers is not in the document]
Comment (kappratiksha) [reply releasetool/commands/tag/java.py]: Done

**Review by chingor13 — APPROVED** [review]

File: releasetool/commands/tag/java.py [hunk 2]
```diff
@@ -55,11 +55,21 @@ def kokoro_job_name(upstream_repo: str, package_name: str) -> Union[str, None]:
     if repo_short_name in java_framework_release_pool_repos:
         return f"cloud-java-frameworks/{repo_short_name}/stage"
 
+    if repo_short_name == "functions-framework-java" and package_name in functions_framework_java_packages:
+        if package_name == "java-function-invoker":
+            return f"functions-framework/java/invoker/release"
```
Comment (chingor13) [inline releasetool/commands/tag/java.py:60]: nit: this seems a bit odd to have a special case for this single component vs. naming the job to match the component

Comment (kappratiksha) [reply releasetool/commands/tag/java.py:60]: Changing the job name

**Review by chingor13 — CHANGES_REQUESTED** [review]
Existing tests are also failing. You can run my running `nox` from the root directory.

**Comment by kappratiksha** [comment]
> Existing tests are also failing. You can run my running `nox` from the root directory.

Fixed the tests.

**Comment by kappratiksha** [comment]
> > Existing tests are also failing. You can run my running `nox` from the root directory.
> 
> Fixed the tests.

I tested this locally and it works

**Review by jihuin — APPROVED** [review]

**Review by chingor13 — APPROVED** [review]

**Pull request merged**